duckduckgo/iOS
# Autofill In Browser layout improvements for landscape / iPad

Task/Issue URL: https://app.asana.com/0/72649045549333/1202809065048067/f
Tech Design URL:
CC:

**Description**:
Landscape and iPad layout handling for all prompts including:
- Save login
- Save username / password
- Save additional login
- Update username / password
- Choose from saved login (single)
- Choose from saved login (multiple)



**Steps to test this PR**:
1. Run through the different view states confirming the layouts match designs for iPhone landscape and iPad portrait & landscape
2. Confirm iPhone portrait layouts are still correct

Note:
- iOS 14 has not been throughly tested (will do a full iOS 14 review of all Autofill screens separately)
- Custom detents not implemented for iOS 16 (will tackle in an iOS 16 improvements PR)



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
